contracts/contracts/strategies/ConvexStrategy.sol

```diff
@@ -88,10 +80,10 @@ contract ConvexStrategy is BaseCurveStrategy {
         onlyGovernor
```

Comment:
We can ditch this method, and the event for it.

Reply:
good point thanks, removing